RayTraceAntiXray/src/main/java/com/vanillage/raytraceantixray/tasks/UpdateBukkitRunnable.java

```diff
@@ -33,6 +33,11 @@ public UpdateBukkitRunnable(RayTraceAntiXray plugin) {
     public void run() {
         plugin.getServer().getOnlinePlayers().forEach(p -> {
```

Comment:
All the logic inside here should be put into a method with a player parameter in the main class `RayTraceAntiXray`. The method is then invoked for each player in the scheduler for Paper and by the `EntityScheduler`s for Folia. We can remove this class `UpdateBukkitRunnable` then.